build-system/test-configs/config.js

```diff
@@ -263,48 +263,7 @@ const htmlFixtureGlobs = [
   '!examples/amp-script/vue-todomvc.amp.html',
   '!examples/amp-skimlinks.html',
   '!examples/amp-smartlinks.html',
-  '!examples/amp-story/access.html',
-  '!examples/amp-story/ad-development.html',
-  '!examples/amp-story/ads/app-install.html',
-  '!examples/amp-story/affiliate-link.html',
-  '!examples/amp-story/amp-story-animation.html',
-  '!examples/amp-story/amp-story-auto-ads.html',
-  '!examples/amp-story/amp-story-branching.html',
-  '!examples/amp-story/amp-story-desktop-one-panel.html',
-  '!examples/amp-story/amp-story-panning-media.html',
-  '!examples/amp-story/ampconf.html',
-  '!examples/amp-story/analytics.html',
-  '!examples/amp-story/animations-sequence.html',
-  '!examples/amp-story/attachment.html',
-  '!examples/amp-story/auto-analytics.html',
-  '!examples/amp-story/consent-geo.html',
-  '!examples/amp-story/consent.html',
-  '!examples/amp-story/cta-layer-outlink.html',
-  '!examples/amp-story/doubleclick.html',
-  '!examples/amp-story/fake-ad.html',
-  '!examples/amp-story/grid-layer-presets.html',
-  '!examples/amp-story/grid-layer-templates.html',
-  '!examples/amp-story/helloworld.html',
-  '!examples/amp-story/interactive_polls.html',
-  '!examples/amp-story/interactive_quizzes.html',
-  '!examples/amp-story/interactive_results.html',
-  '!examples/amp-story/interactives.html',
-  '!examples/amp-story/player-local-stories.html',
-  '!examples/amp-story/player-story-attribution.html',
-  '!examples/amp-story/player-with-button.html',
-  '!examples/amp-story/player.html',
-  '!examples/amp-story/progress-bar.html',
-  '!examples/amp-story/quiz.html',
-  '!examples/amp-story/rtl.html',
-  '!examples/amp-story/show-tooltip.html',
-  '!examples/amp-story/supports-landscape-ads.html',
-  '!examples/amp-story/text-background-color.html',
-  '!examples/amp-story/video-one-page.html',
-  '!examples/amp-story/video-one-page2.html',
-  '!examples/amp-story/videos-cdn.html',
-  '!examples/amp-story/videos-google-cache.html',
-  '!examples/amp-story/videos.html',
-  '!examples/amp-story/visual-effects.html',
+  '!examples/{,visual-tests/}amp-story/**/*.html',
```

Comment:
Two questions:
1. If these aren't meant to be valid AMP, they should be moved from this "to fix" list to the "always invalid" list above.
2. Why are we including `visual-tests`? IIUC, all visual tests are run against valid AMP pages. Unless I'm wrong?

Reply:
I based the glob suggestion on the fact that there many existing visual-diff examples of amp-story. Not sure if that's intended or not.

Reply:
On the same lines, there are E2E tests for stories in the ignore list as well. Should they be ignore too?

Reply:
I don't see any visual tests in this list, so we can not include them in this TODO list. Then adding the `examples/amp-story` to the invalid docs sounds good.

Reply:
In general I see a couple of things that don't make sense to have in this list, so I can check those things and then re-request the review. For instance, `amp-story-player` documents are not meant to be AMP documents in many cases, as the player works on non-AMP docs

Reply:
Perhaps even better, `foobar.invalid.html` naming convention?

Reply:
I'm pretty sure `globby` (which uses `fast-glob`) does not support positive or negative globs for non-matches (e.g. all files that _don't_ end with `.amp.html`). Using a `invalid.html` convention SGTM. 👍

Reply:
Globby does support excluding matches. I'm not sure exactly what you're trying to do, but ex. 
`globby.sync(['examples/**/*.html', '!examples/**/*.amp.html'])` works as you'd expect

Reply:
Yes, but I don't think it supports excluding non-matches. E.g. _"ignore every file under `foo/` directory that does not match `bar` pattern"_.

Reply:
I made it work already by excluding `<path>/!(*.amp.html)`, so essentially exclude the amp.html from the exclusion. Will update the PR soon.